student-store-express-api/models/StoreModel.js

```diff
@@ -0,0 +1,64 @@
+const { storage } = require("../data/storage");
+
+class StoreModel {
+  static async listOrders() {
+    const purchases = storage.get("purchases").value();
+    return purchases;
+  }
+  static async listProducts() {
+    const products = storage.get("products").value();
+    return products;
+  }
+  static async fetchProductById(productId) {
+    // fetch a single product
+    const product = storage
+      .get("products")
+      .find({ id: Number(productId) })
+      .value();
+    return product;
+  }
+  static async fetchOrderById(orderId) {
+    // fetch a single order
+    const order = storage
+      .get("purchases")
+      .find({ id: Number(orderId) })
+      .value();
+    return order;
+  }
+  static async recordOrder(order) {
+    // create a new order
+
```

Comment:
Nitpick: rm whitespace
```suggestion
    // create a new order
```